package.json

```diff
@@ -83,6 +83,7 @@
 		"@types/react": "16.9.35",
 		"@types/wordpress__data": "4.6.7",
 		"@types/wordpress__element": "2.4.1",
+		"@woocommerce/woocommerce-rest-api": "^1.0.1",
```

Comment:
Probably want to pin this version (remove `^`) - RenovateBot will pin it anyway, save a PR.